model/event/subscriptions.go

```diff
@@ -50,28 +48,77 @@ type Subscription struct {
 	Subscriber     Subscriber    `bson:"subscriber"`
 	Owner          string        `bson:"owner"`
 	OwnerType      OwnerType     `bson:"owner_type"`
+	ExtraData      interface{}   `bson:"extra_data,omitempty"`
+}
+
+type unmarshalSubscription struct {
+	ID             bson.ObjectId `bson:"_id"`
+	Type           string        `bson:"type"`
+	Trigger        string        `bson:"trigger"`
+	Selectors      []Selector    `bson:"selectors,omitempty"`
+	RegexSelectors []Selector    `bson:"regex_selectors,omitempty"`
+	Subscriber     Subscriber    `bson:"subscriber"`
+	Owner          string        `bson:"owner"`
+	OwnerType      OwnerType     `bson:"owner_type"`
+	ExtraData      bson.Raw      `bson:"extra_data,omitempty"`
```

Comment:
can we serialize this (and the objectID above, in a way that doesn't require the type to be bson.Raw?